### PR TITLE
cli: adds mode and file for docs

### LINKED ISSUE #2096: fixtures: insert/replace mode for docs
it would be useful to have a quick way to update docs on running PROD instance.

Currently nothing much is possible:

```console
$ cernopendata fixtures docs --help
Usage: cernopendata fixtures docs [OPTIONS]

  Load demo article records.

Options:
  --help  Show this message and exit.
```

Ideally we'd like to be able to update docs in a similar way as records:

```console
$ cernopendata fixtures records --help
Usage: cernopendata fixtures records [OPTIONS]

  Load all records.

Options:
  --skip-files                    Skip loading of files
  -f, --file PATH                 Path to the file(s) to be loaded. If not
                                  provided, allfiles will be loaded
  --profile                       Output profiling information.
  --verbose
  --mode [insert|replace|insert-or-replace]
                                  [required]
  --help                          Show this message and exit.
``` 

@ioannistsanaktsidis When time permits, can you please introduce the `-f` option and the `--mode` option with insert/replace possibilities for docs fixtures as well? 

We have `docid`'s and `slug`'s that we can profit from...

(stemmed from point no.3 of https://github.com/cernopendata/opendata.cern.ch/pull/1887#issuecomment-348214957)

### PR DESCRIPTION
**Pull request by ioannistsanaktsidis**
* Closes #2096.

***
 - cernopendata fixtures docs -f /code/cernopendata/modules/fixtures/data/docs/alice-about/alice-about.json 

- cernopendata fixtures docs -f /code/cernopendata/modules/fixtures/data/docs/alice-about/alice-about.json --mode replace
***

Signed-off-by: Ioannis Tsanaktsidis <ioannis.tsanaktsidis@cern.ch>

### CODE REVIEW

**Comment by tiborsimko** [comment]
Note the CI (cosmetic) failures...


```
=================================== FAILURES ===================================

__________________________________ PEP8-check __________________________________

/code/cernopendata/modules/fixtures/cli.py:334:80: E501 line too long (85 > 79 characters)

                                                                    data['title']))))

                                                                               ^

/code/cernopendata/modules/fixtures/cli.py:339:80: E501 line too long (89 > 79 characters)

                                str(slugify(data.get('slug', data['title'])))), err=True)

                                                                               ^

===================== 1 failed, 63 passed in 1.67 seconds ======================
```

File: cernopendata/modules/fixtures/cli.py [hunk 1]
```diff
@@ -252,14 +271,21 @@ def glossary_terms():
 
 
 @fixtures.command()
+@click.option('files', '--file', '-f', multiple=True,
+              type=click.Path(exists=True),
+              help='Path to the file(s) to be loaded. If not provided, all'
+                   'files will be loaded')
+@click.option('--mode', required=True, type=click.Choice(
+    ['insert', 'replace']))
```
Comment (tiborsimko) [inline cernopendata/modules/fixtures/cli.py]: insert-or-replace may be good for harmony with `records` fixtures

**Comment by ioannistsanaktsidis** [comment]
@tiborsimko  updated with the latest changes.

**Comment by tiborsimko** [comment]
@ioannistsanaktsidis The CLI options regarding insertion or replace seem to work OK, but the docs does not seem to be fully updated everywhere. E.g. I see an updated version on the search results page, but not on the detailed record page. Is the DB commit and the ES indexer properly called, so that the docs metadata and markdown body changes are propagated well to both DB and ES? Here's a test case scenario:

```
docker-compose down -v
docker-compose -f docker-compose-dev.yml down -v
docker-compose -f docker-compose-dev.yml build
docker-compose -f docker-compose-dev.yml up -d
sleep 20
docker exec -i -t opendatacernch_web_1 /code/scripts/populate-instance.sh --skip-records
wget -q --spider http://0.0.0.0:5000/docs/cms-getting-started-2010
wget -q --spider http://0.0.0.0:5000/api/docs/cms-getting-started-2010
wget -q -O /tmp/z_OLD_html http://0.0.0.0:5000/docs/cms-getting-started-2010
wget -q -O /tmp/z_OLD_api http://0.0.0.0:5000/api/docs/cms-getting-started-2010
docker exec -i -t opendatacernch_web_1 sed -i 's,CMS,XXXCMSXXX,g' /code/cernopendata/modules/fixtures/data/docs/cms-getting-started-2010/cms-getting-started-2010.json
docker exec -i -t opendatacernch_web_1 sed -i 's,CMS,XXXCMSXXX,g' /code/cernopendata/modules/fixtures/data/docs/cms-getting-started-2010/cms-getting-started-2010.md
docker exec -i -t opendatacernch_web_1 cernopendata fixtures docs -f /code/cernopendata/modules/fixtures/data/docs/cms-getting-started-2010/cms-getting-started-2010.json --mode replace
wget -q --spider http://0.0.0.0:5000/docs/cms-getting-started-2010
wget -q --spider http://0.0.0.0:5000/api/docs/cms-getting-started-2010
wget -q -O /tmp/z_NEW_html http://0.0.0.0:5000/docs/cms-getting-started-2010
wget -q -O /tmp/z_NEW_api http://0.0.0.0:5000/api/docs/cms-getting-started-2010
diff -q /tmp/z_OLD_html /tmp/z_NEW_html
diff -q /tmp/z_OLD_api /tmp/z_NEW_api
```

**Comment by ioannistsanaktsidis** [comment]
@tiborsimko  problem solved. I think now it is ready to go.

**Pull request merged**